src/reference/configuration/listening_address.md

```diff
@@ -30,3 +30,44 @@ Open the Connection Manager and proceed with the following steps:
 
 2. Select the appropriate radio button. _If using a_ `Custom` _listening address - supply the address in the input field._
 3. Supply the value of the desired listening port.
+
+## Proxying Local Traffic
```

Comment:
I think this should be a guide: `How to proxy local traffic?`
In this file we can reference to it though.